src/main/java/org/jenkinsci/test/acceptance/po/CapybaraPortingLayerImpl.java

```diff
@@ -344,6 +344,8 @@ protected static boolean isStale(WebElement element) {
             return false;
         } catch (StaleElementReferenceException expected) {
             return true;
+        } catch (NoSuchElementException expected) {
```

Comment:
as I understand this, this exception should be impossible.  
in order to be here (given the stack) the `WebElement` has been interacted with (via `click`) so we know it exists at that point in time.

all we are doing is a freshess check which according to the Javadoc (which may be lying) is

> All method calls will do a freshness check to ensure that the element reference is still valid. This essentially determines whether or not the element is still attached to the DOM. If this test fails, then an StaleElementReferenceException is thrown, and all future calls to this instance will fail.

thus it should not even be trying to re-resolve the `WebElement` , but just saying yup it is no longer in the DOM, and throw a `StaleElementReferenceException`
`NoSuchElementExcetion` should only be throw when `findElement` is called (either in `WebDriver` or `WebElement`)

Reply:
I checked my docker images and it was working previously with Firefox 80, and got broken in Firefox 83.

Looking at the Firefox 82 [changelog](https://developer.mozilla.org/en-US/docs/Mozilla/Firefox/Releases/82), I can find this of interest

> Element.setPointerCapture() throws the NotFoundError exception if the pointer id is invalid, as per the specification (bug 1662124). Previously it wrongly threw an InvalidPointerId exception.

Which AFAIU could translate into `StaleElementReferenceException` vs. `NoSuchElementException` on selenium side.

Reply:
so this sounds like a bug in Selenium?  ie it should now translate this when not using `findBy`?

I'm hesitant to make this change as there will likely be a lot of other places that also do something similar.

Reply:
https://github.com/SeleniumHQ/selenium/issues/8929